integrations/elasticsearch/elasticsearch-events/sensu-resources.yaml

```diff
@@ -0,0 +1,106 @@
+---
+type: Pipeline
+api_version: core/v2
+metadata:
+  name: elasticsearch
+  labels:
+    provider: events
+spec:
+  workflows:
+  - name: elasticsearch-events
+    filters: []
+    mutator: {}
+    handler:
+      api_version: core/v2
+      type: Handler
+      name: elasticsearch-events
+---
+type: Handler
+api_version: core/v2
+metadata:
+  name: elasticsearch-events
+spec:
+  command: >-
+    sensu-elasticsearch-handler
+    --index $ELASTICSEARCH_INDEX
+    --index-rotation daily
+    --omit-redundant-fields
+    --url $ELASTICSEARCH_URL
```

Comment:
```suggestion
```

No need to explicitly set this flag since [you're setting the environment variable](https://github.com/sensu/catalog/pull/151/files#diff-21f1febc19271950dfb84121b479eb5d750976c62084c9f24738d1a99c6974d2R30) (which is [supported by the handler](https://github.com/sensu/sensu-elasticsearch-handler#environment-variables)).